tsdb/isolation.go

```diff
@@ -307,6 +307,9 @@ type txRingIterator struct {
 }
 
 func (it *txRingIterator) At() uint64 {
+	if int(it.pos) >= len(it.ids) {
```

Comment:
As discussed on the PR level, we don't need isolation for ooo chunks, so I would not expect a change here, can we get rid of this?

Reply:
Done in 881bda67535b858e0f4f2e0ede0df9eab667098f.